File: scripts/javascript/billion.js
let billion = 1000000000;
let start_time = Date.now();

for (let i = 0; i < billion; i++) {
}

let end_time = Date.now();
let elapsed_time = (end_time - start_time) / 1000;
console.log(`${elapsed_time.toFixed(3)} segundos`);

File: scripts/rust/billion.rs
use std::time::Instant;

fn main() {
    let billion = 1_000_000_000;
    let start_time = Instant::now();

    for _ in 0..billion {
    }

    let end_time = Instant::now();
    let elapsed_time = end_time.duration_since(start_time).as_secs_f64();
    println!("{:.3} segundos", elapsed_time);
}
